compose-settings-storage-base/src/commonMain/kotlin/com/alorma/compose/settings/storage/base/ValueProviders.kt

```diff
@@ -2,9 +2,11 @@ package com.alorma.compose.settings.storage.base
 
 import kotlin.reflect.KProperty
 
+@Deprecated(message = "To be deleted")
```

Comment:
Why were these deprecated? I didn't see anything in the release notes

Reply:
Will publish a 2.0 version soon.

And improve the Readme too, with all the examples.

Basically... The `val state by rememberSomething()` won't be needed anymore.

As setting tiles composables won't handle the state for themselves, we can use simpler:

```
val state = rememberSomething()

state.value // To read
state.value = "..." // To write
```